test/NuGet.Core.Tests/NuGet.Packaging.Core.Test/NuSpecCoreReaderTests.cs

```diff
@@ -38,17 +59,19 @@ public void GetPackageType_ReturnsEmptyPackageTypeListIfNotSpecifiedInManfiest()
    <somestuff>some-value</somestuff>
    <ver>123</ver>
   </metadata>
-</package>")]
+</package>")] // Elements with no children should be extracted.
         [InlineData(@"<?xml version=""1.0""?>
 <package xmlns=""a-random-xsd"">
   <metadata>
-   <packageType name=""Managed"" version=""2.0"" />
+   <packageTypes>
```

Comment:
Nope. This feature (quirks mode) was never fully implemented and packages were never stamped with `<packageType>`. In fact, the existing code coalesces no `<packageType>` element "legacy" which is the only acceptable value.

Furthermore, the `<packageType>` element has already changed (the `name` attribute used to be the text content rather than an attribute).